discord_bot/discord_bot/add_roles.py

```diff
@@ -11,6 +11,7 @@
 from dtower.tourney_results.constants import champ, copper, gold, leagues, plat, silver
 from dtower.tourney_results.data import get_results_for_patch, get_tourneys
 from dtower.tourney_results.models import PatchNew as Patch
+import const
```

Comment:
`from discord_bot import const`

otherwise it won't work unless you're running the code in this specific directory, which we don't guarantee to do